opencv/cvat
# Remove a superflous encodeURI() to avoid double encoding non latin paths - Fix for the issue #4428: Cannot connect file share，cannot recognize non-ascii path #4428

### Motivation and context


Fixing the issue non-ascii-path #4428
When you create a task and trying to select files from `Connected file share`,
you get multiple `Could not load share data from the server` error popups
if you try to expand a directory named in non-ascii code.

### How has this been tested?


Following documentation https://opencv.github.io/cvat/docs/administration/basics/installation/#share-path,
setup a share path for CVAT, which contains (sub)directory encoded in non ascii code.

- Create a project and task, and then `Connected file share` tab under `Select files` section.
- Click/expand directories until you encounter the non-ascii named directory.
- Expand the non-ascii named directory and see that no error dialog pop-ups.

### Checklist

- ~~[ ] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file~~
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly~~
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- ~~[ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))~~

### License

  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
